docs/section-7/control.md

```diff
@@ -8,13 +8,19 @@ Or is there?
 
 Control systems are the solutions to the three problems mentioned above. They help programmers manipulate motors to a great deal of control and flexibility. When designing a control algorithm for a robot mechanism, there are a number of different approaches to take. These range from very simple approaches, to advanced and complex ones. Each has tradeoffs. Some will work better than others in different situations, some require more mathematical analysis than others.
 
-There are two fundamental types of mechanism controller that we will cover here:
+There are two fundamental types of mechanism controllers that we will cover here:
```

Comment:
provide a short and easy-to understand TL;DR, please. It's very useful.

something like,
"Feed**Forward**" is for predicting how you will act, feed**back** is for fixing your mistakes.

Reply:
done